packages/lesswrong/components/posts/AddToCalendar/AddToCalendarIcon.tsx

```diff
@@ -0,0 +1,67 @@
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import React from 'react';
+import moment from '../../../lib/moment-timezone';
+import Add from '@material-ui/icons/Add';
+import AddToCalendar from '@culturehq/add-to-calendar';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  plusIcon: {
+    position: 'absolute',
+    top: 11,
+    left: 11,
+    height: 10,
+    width: 10,
+    backgroundColor: 'white'
+  },
+  label: {
+    marginLeft: 3
+  }
+})
+
+const AddToCalendarIcon = ({post, label, hideTooltip, classes}: {
+  post: PostsBase,
+  label?: string,
+  hideTooltip?: boolean,
+  classes: ClassesType,
+}) => {
+  // need both a start and end time to add an event to a calendar
+  if (!post.startTime || !post.endTime) {
+    return null;
+  }
+  
+  const calendarIconNode = (
+    <AddToCalendar
+      event={{
+        name: post.title,
+        details: post.facebookLink,
```

Comment:
I'd love to put an event description in here instead, but I don't think google would like the post body as-is. It might actually make sense to have a separate field for a "simplified" event description, to use here as well as other places with less space, like the popup in the map.